components/prism-batch.js

```diff
@@ -76,7 +76,7 @@
 			},
 			{
 				// Other commands
-				pattern: /((?:^|[&(])[ \t]*@?)\w+\b(?:[^^&)\r\n]|\^(?:\r\n|[\s\S]))*/im,
+				pattern: /((?:^|[&(])[ \t]*@?)\w+\b(?:"(?:[\\"]"|[^"])*"|[^^&)\r\n]|\^(?:\r\n|[\s\S]))*/im,
```

Comment:
For similar reasons as mentioned above, this might cause exponential backtracking. This is how to fix it.

```suggestion
				pattern: /((?:^|[&(])[ \t]*@?)\w+\b(?:"(?:[\\"]"|[^"])*"(?!")|[^"^&)\r\n]|\^(?:\r\n|[\s\S]))*/im,
```